tests/utils.js

```diff
@@ -899,6 +921,7 @@ module.exports = {
   revertDbNative,
 
   resetBrowser: () => {
+    deprecated('utils.resetBrowser', 'utils.resetBrowserNative');
```

Comment:
This function is not deprecated anymore.

Reply:
Done